src/app/components/Account/TokenPills.tsx

```diff
@@ -50,7 +51,11 @@ export const Pill: FC<PillProps> = ({ pill }) => {
       component={Link}
       href={href}
       key={pill.token_contract_addr}
-      label={`${pill.balance} ${pill.token_name}`}
+      label={
+        <>
+          <RoundedBalance value={pill.balance} /> {pill.token_name}
```

Comment:
Oops, it now says: `337325.43837 ROSE FTP, -337295.51 ROSE YUZU`
instead of saying: `337325.43837 FTP, -337295.51 YUZU`

Reply:
🤦 will fix that

Reply:
https://github.com/oasisprotocol/explorer/pull/174